src/aiab/catch.mc

```diff
@@ -102,12 +102,11 @@ function catch {
 
   # Find the allay the player is looking at
   tag @s add aiab.this0
-  execute as @e[type=minecraft:allay,distance=..5,limit=1,sort=nearest] run {
+  execute as @e[type=#aiab:catch/aiabmobs,distance=..5,limit=1,sort=nearest] run {
```

Comment:
aiab of aiabmobs could be omitted, since the namespace is already there
```suggestion
  execute as @e[type=#aiab:catch/mobs,distance=..5,limit=1,sort=nearest] run {
```